docs/best-practices/gold-way-work.md

```diff
@@ -130,6 +130,8 @@ import BestPracticeSvg from './images/best_practice.svg';
       <li>Automated testing (we recommend cypress)</li>
       <li>Visit our Example Repo: <a href="https://github.com/bcgov/keycloak-example-apps" target="_blank">https://github.com/bcgov/keycloak-example-apps</a></li>
    </ul>
+   <h1>Common Questions</h1>
+   <li>Visit our Example Repo: <a href="https://stackoverflow.developer.gov.bc.ca/questions/tagged/146" target="_blank">Visit here for some FAQs on Custom Realms</a></li>
```

Comment:
Should the visit our example repo part here be removed?

Reply:
@jlangy very good catch...updated so let me know if it came through